src/app/Models/Setting.php

```diff
@@ -4,11 +4,34 @@
 
 use Backpack\CRUD\CrudTrait;
 use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Facades\Storage;
+use Intervention\Image\Facades\Image;
 
 class Setting extends Model
 {
     use CrudTrait;
 
     protected $table = 'settings';
     protected $fillable = ['value'];
+
+
+    public function setValueAttribute($value)
+    {
+        $attribute_name = "value";
```

Comment:
If that `$value` above is the attribute name, why not call it attribute name and refactor as appropriate?